qa/tasks/restart.py

```diff
@@ -3,6 +3,7 @@
 """
 import logging
 import pipes
+import os
```

Comment:
@lxbsz Could you undo change to this source? PR #49423 removes this file since its obsolete.

Reply:
Never mind. I'll merge this (its being lying around and is tested) and then the other PR.